<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsFile</class>
 <widget class="QWidget" name="SettingsFile">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>756</width>
    <height>617</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin">
    <number>8</number>
   </property>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Select what file types you wish to associate with Black Omega.</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="m_fileTable">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>325</height>
      </size>
     </property>
     <row>
      <property name="text">
       <string/>
      </property>
     </row>
     <row>
      <property name="text">
       <string/>
      </property>
     </row>
     <row>
      <property name="text">
       <string/>
      </property>
     </row>
     <row>
      <property name="text">
       <string/>
      </property>
     </row>
     <row>
      <property name="text">
       <string/>
      </property>
     </row>
     <row>
      <property name="text">
       <string/>
      </property>
     </row>
     <row>
      <property name="text">
       <string/>
      </property>
     </row>
     <row>
      <property name="text">
       <string/>
      </property>
     </row>
     <row>
      <property name="text">
       <string/>
      </property>
     </row>
     <row>
      <property name="text">
       <string/>
      </property>
     </row>
     <column>
      <property name="text">
       <string>File Extension</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Description</string>
      </property>
     </column>
     <item row="0" column="0">
      <property name="text">
       <string>.flac</string>
      </property>
      <property name="checkState">
       <enum>Unchecked</enum>
      </property>
      <property name="flags">
       <set>ItemIsSelectable|ItemIsUserCheckable|ItemIsEnabled</set>
      </property>
     </item>
     <item row="0" column="1">
      <property name="text">
       <string>Free Lossless Audio Codec</string>
      </property>
      <property name="flags">
       <set>ItemIsSelectable|ItemIsUserCheckable|ItemIsEnabled</set>
      </property>
     </item>
     <item row="1" column="0">
      <property name="text">
       <string>.mp3</string>
      </property>
      <property name="checkState">
       <enum>Unchecked</enum>
      </property>
      <property name="flags">
       <set>ItemIsSelectable|ItemIsUserCheckable|ItemIsEnabled</set>
      </property>
     </item>
     <item row="1" column="1">
      <property name="text">
       <string>MPEG Audio Layer-3</string>
      </property>
      <property name="flags">
       <set>ItemIsSelectable|ItemIsUserCheckable|ItemIsEnabled</set>
      </property>
     </item>
     <item row="2" column="0">
      <property name="text">
       <string>.m4a</string>
      </property>
      <property name="checkState">
       <enum>Unchecked</enum>
      </property>
      <property name="flags">
       <set>ItemIsSelectable|ItemIsUserCheckable|ItemIsEnabled</set>
      </property>
     </item>
     <item row="2" column="1">
      <property name="text">
       <string>MPEG-4 Audio File</string>
      </property>
      <property name="flags">
       <set>ItemIsSelectable|ItemIsUserCheckable|ItemIsEnabled</set>
      </property>
     </item>
     <item row="3" column="0">
      <property name="text">
       <string>.m4b</string>
      </property>
      <property name="checkState">
       <enum>Unchecked</enum>
      </property>
      <property name="flags">
       <set>ItemIsSelectable|ItemIsUserCheckable|ItemIsEnabled</set>
      </property>
     </item>
     <item row="3" column="1">
      <property name="text">
       <string>MPEG-4 Audiobook File</string>
      </property>
      <property name="flags">
       <set>ItemIsSelectable|ItemIsUserCheckable|ItemIsEnabled</set>
      </property>
     </item>
     <item row="4" column="0">
      <property name="text">
       <string>.ogg</string>
      </property>
      <property name="checkState">
       <enum>Unchecked</enum>
      </property>
      <property name="flags">
       <set>ItemIsSelectable|ItemIsUserCheckable|ItemIsEnabled</set>
      </property>
     </item>
     <item row="4" column="1">
      <property name="text">
       <string>Vorbis Ogg</string>
      </property>
      <property name="flags">
       <set>ItemIsSelectable|ItemIsUserCheckable|ItemIsEnabled</set>
      </property>
     </item>
     <item row="5" column="0">
      <property name="text">
       <string>.wav</string>
      </property>
      <property name="checkState">
       <enum>Unchecked</enum>
      </property>
      <property name="flags">
       <set>ItemIsSelectable|ItemIsUserCheckable|ItemIsEnabled</set>
      </property>
     </item>
     <item row="5" column="1">
      <property name="text">
       <string>Waveform Audio File</string>
      </property>
      <property name="flags">
       <set>ItemIsSelectable|ItemIsUserCheckable|ItemIsEnabled</set>
      </property>
     </item>
     <item row="6" column="0">
      <property name="text">
       <string>.pls</string>
      </property>
      <property name="checkState">
       <enum>Unchecked</enum>
      </property>
      <property name="flags">
       <set>ItemIsSelectable|ItemIsUserCheckable|ItemIsEnabled</set>
      </property>
     </item>
     <item row="6" column="1">
      <property name="text">
       <string>Generic Playlist File</string>
      </property>
      <property name="flags">
       <set>ItemIsSelectable|ItemIsUserCheckable|ItemIsEnabled</set>
      </property>
     </item>
     <item row="7" column="0">
      <property name="text">
       <string>.m3u</string>
      </property>
      <property name="checkState">
       <enum>Unchecked</enum>
      </property>
      <property name="flags">
       <set>ItemIsSelectable|ItemIsUserCheckable|ItemIsEnabled</set>
      </property>
     </item>
     <item row="7" column="1">
      <property name="text">
       <string>MP3 Playlist File</string>
      </property>
      <property name="flags">
       <set>ItemIsSelectable|ItemIsUserCheckable|ItemIsEnabled</set>
      </property>
     </item>
     <item row="8" column="0">
      <property name="text">
       <string>.m3u8</string>
      </property>
      <property name="checkState">
       <enum>Unchecked</enum>
      </property>
      <property name="flags">
       <set>ItemIsSelectable|ItemIsUserCheckable|ItemIsEnabled</set>
      </property>
     </item>
     <item row="8" column="1">
      <property name="text">
       <string>MP3 Unicode Playlist File</string>
      </property>
      <property name="flags">
       <set>ItemIsSelectable|ItemIsUserCheckable|ItemIsEnabled</set>
      </property>
     </item>
     <item row="9" column="0">
      <property name="text">
       <string>.xspf</string>
      </property>
      <property name="checkState">
       <enum>Unchecked</enum>
      </property>
      <property name="flags">
       <set>ItemIsSelectable|ItemIsUserCheckable|ItemIsEnabled</set>
      </property>
     </item>
     <item row="9" column="1">
      <property name="text">
       <string>XML Shareable Playlist Format</string>
      </property>
      <property name="flags">
       <set>ItemIsSelectable|ItemIsUserCheckable|ItemIsEnabled</set>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="m_explorerGroup">
     <property name="title">
      <string>Default behavior when opening file</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QRadioButton" name="m_expPlayRadio">
        <property name="text">
         <string>Play file immediately</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="m_expQueueRadio">
        <property name="text">
         <string>Queue file at end of playlist</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="m_directoryCheckbox">
     <property name="text">
      <string>Associate Black Omega with Directories</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>144</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
